pkg/controller/wildflyserver/wildflyserver_controller_test.go

```diff
@@ -101,7 +102,7 @@ func TestEnvUpdate(t *testing.T) {
 		},
 		Spec: wildflyv1alpha1.WildFlyServerSpec{
 			ApplicationImage: applicationImage,
-			Size:             replicas,
+			Size:             0,
```

Comment:
why do you create the app without any pods?

Reply:
The test does not need any pod for check it works. The reason why it does not work is that the operator waits for the all pods to up and running before it permits other operations to be processed.
The reconcile loop is defined here:
https://github.com/wildfly/wildfly-operator/pull/54/files#diff-d96e34894c3a7c83e1eb1ad11a0ab147R224
The handling is here because of the recovery processing lists all the available pods and then  works above it.